spec/audited/sweeper_spec.rb

```diff
@@ -1,6 +1,12 @@
 require "spec_helper"
 
 class AuditsController < ActionController::Base
+  if Rails::VERSION::MAJOR == 4
+    before_filter :populate_user
```

Comment:
`before_action` already exists on rails 4.0
https://github.com/rails/rails/blob/f53132deea65ae673b29b811c606be103f1ae3ab/actionpack/lib/abstract_controller/callbacks.rb
I would suggest to remove this `if` here